plugin/evm/vm.go

```diff
@@ -144,39 +144,22 @@ const (
 	txGossipBloomMaxItems          = 8 * 1024
 	txGossipBloomFalsePositiveRate = 0.01
 	txGossipMaxFalsePositiveRate   = 0.05
-	txGossipTargetResponseSize     = 20 * units.KiB
+	txGossipMaxSize                = 20 * units.KiB
```

Comment:
I don't think `maxSize` is correct - we allow larger sizes.

Reply:
renamed to txGossipTargetMessageSize